components/movementsensor/replay/replay_test.go

```diff
@@ -210,9 +226,7 @@ func TestReplayMovementSensorFunctions(t *testing.T) {
 				LocationID:     validLocationID,
 				OrganizationID: validOrganizationID,
 			},
-			methods:      []method{linearAcceleration},
-			startFileNum: map[method]int{linearAcceleration: -1},
-			endFileNum:   map[method]int{linearAcceleration: -1},
```

Comment:
`methods` are now unused, since we're looping over all methods. `startFileNum` and `endFileNum` are not necessary here anymore. Setting them to `-1` means we don't want to loop over the method. The same behavior is now achieved differently in the test, without the need to set the file indices to `-1`.